bin/perl/translate-module-info.pl

```diff
@@ -13,12 +13,14 @@
 use Importer 'Code::Tooling::Util::JSON' => qw[ decode ];
 
 sub main {
-    my $file = Path::Class::File->new( 'report.json' );
-    my $content = $file->slurp;
+    my $content = '';
+    for my $line ( <STDIN> ) {
+        $content .= $line;
+    }
```

Comment:
There is no need to loop here, please look at the scripts in the `bin/json` directories for examples of how to do this.